docs/en/2_new_data_model.md

```diff
@@ -20,8 +20,7 @@ Usually we recommend to use the first two methods which are usually easier than
 
 In this note, we give an example for converting the data into COCO format.
 
-**Note**: MMDetection only supports evaluating mask AP of dataset in COCO format for now.
-So for instance segmentation task users should convert the data into coco format.
+**Note**: Datasets and metrics have been decoupled except CityScapes since MMDetection 3.0 . So users can evaluate on VOC dataset with COCO metric, or evaluate on VOC dataset with both VOC and COCO metrics.
```

Comment:
The coco dataset should also be able to use the voc metric, and many users still use it like this